compiler/src/dotty/tools/dotc/typer/ErrorReporting.scala

```diff
@@ -275,7 +276,7 @@ object ErrorReporting {
       else
         val add = suggestImports(
           ViewProto(qualType.widen,
-            SelectionProto(tree.name, WildcardType, NoViewsAllowed, privateOK = false)))
+            SelectionProto(tree.name, WildcardType, NoViewsAllowed, privateOK = false, NoSpan)))
```

Comment:
Is it wrong if this uses tree.nameSpan as well?

Reply:
It's a `RefTree` (it doesn't even have a `nameSpan`), so if anything `tree.span`. But this is for auto import suggestion, so `NoSpan` may actually be more closer to truth. What do you think?

Reply:
Makes sense, of course. 👍🏼